File: C#/Урок 042 - break/Задача 01.cs
class Program
{
    static void Main()
    {
        while (true)
        {
            System.Console.Write("Enter a number (0 to stop): ");
            int number = System.Convert.ToInt32(System.Console.ReadLine());

            if (number == 0)
                break;

            System.Console.WriteLine($"{number}^2 = {number * number}");
        }
    }
}

File: C++/Урок 033 - do while/Задача 01.cpp
#include <iostream>

int main()
{
    int number;

    do
    {
        std::cout << "Enter a number (0 to stop): ";
        std::cin >> number;

        if (number != 0)
        {
            std::cout << number << " is " << (number % 2 == 0 ? "even" : "odd") << "\n";
        }
    } while (number != 0);

    return 0;
}
